lib/service/database.js

```diff
@@ -321,7 +325,8 @@ DatabaseService.prototype.queueAnalysisOperations = function(analysis, callback)
                     query: annotateNodeType(nodeToUpdate) + transactionQuery([
                         deleteFromCacheTableQuery(targetTableName),
                         populateCacheTableQuery(targetTableName, nodeToUpdate.sql()),
-                        checkCacheTableQuery(targetTableName)
+                        checkCacheTableQuery(targetTableName),
+                        analyzeTableQuery(targetTableName)
```

Comment:
My point of view is pretty similar, we could take care of it later if it's worth it. I was just wondering because I didn't know how it is feasible to fail. IMO, we should not be aware of _corner cases_ (at the moment).